packages/ui/src/components/Chip/Chip.stories.tsx

```diff
@@ -47,6 +47,9 @@ const Template: StoryFn<typeof Chip> = (args) => {
 };
 
 export const NoDismiss = Template.bind({});
+NoDismiss.args = {
+  onDismiss: undefined,
+};
```

Comment:
Leaving args as undefined lead to `onDismiss` being explicitly set to some action :woman_shrugging: